packages/amplify-graphql-auth-transformer/src/resolvers/subscriptions.ts

```diff
@@ -157,7 +202,11 @@ export const generateAuthExpressionForSubscriptions = (providers: ConfiguredAuth
     totalAuthExpressions.push(
       iff(
         equals(ref('util.authType()'), str(COGNITO_AUTH_TYPE)),
-        compoundExpression([...generateStaticRoleExpression(cognitoStaticRoles), ...dynamicRoleExpression(cognitoDynamicRoles)]),
+        compoundExpression([
+          set(ref(HAS_VALID_OWNER_ARGUMENT_FLAG), bool(false)),
```

Comment:
Could this be moved inside `dynamicRoleExpression` block as this would be required only for dynamic roles?